assets/scss/modules/typography/_lists.scss

```diff
@@ -1,17 +1,17 @@
 ul, ol {
   margin: $leading 0 $leading*2 0;
-  padding-left: 1em;
-  list-style-type: square;
+  padding-left: 0px;
+  list-style-type: none;
   letter-spacing: 0.01em;
   list-style-position: inside;
```

Comment:
Should be `outside` I think. If a list item ever wraps over more than one line, the bullet will be visually lost...